src/overlays/actors/ovl_En_Snowwd/z_en_snowwd.c

```diff
@@ -79,23 +71,27 @@ void EnSnowwd_Destroy(Actor* thisx, PlayState* play) {
 }
 
 void func_80AF76F0(EnSnowwd* this, PlayState* play) {
+    static Vec3f accel = { 0.0f, 0.0f, 0.0f };
+    static Vec3f velocity = { 0.0f, -4.0f, 0.0f };
+    static Color_RGBA8 primColor = { 255, 255, 255, 255 };
+    static Color_RGBA8 envColor = { 200, 200, 220, 0 };
     s32 pad;
     Actor* thisx = &this->actor;
-    f32 sp54;
-    Vec3f sp48;
+    f32 wobbleAmplitude;
+    Vec3f pos;
 
     if (this->collider.base.acFlags & AC_HIT) {
         this->collider.base.acFlags &= ~AC_HIT;
         Actor_PlaySfx(thisx, NA_SE_IT_REFLECTION_WOOD);
     }
     if (thisx->home.rot.y != 0) {
-        this->unk190 = 0x15;
+        this->timer = 21;
         thisx->home.rot.y = 0;
         if (thisx->home.rot.z == 0) {
             if (SNOWWD_GET_PARAM_F80(&this->actor) < 0x10) {
```

Comment:
```suggestion
            if (SNOWWD_GET_DROP_TABLE(&this->actor) < DROP_TABLE_SIZE) {
```
Personally this makes sense to me. May want to get a second opinion before implementing it though. 
Note the `DROP_TABLE_SIZE` would apply to the `0x10` multiplier below in the `Item_DropCollectibleRandom` call.